_translator-files/po/de/Running-a-Server.po

```diff
@@ -172,16 +176,20 @@ msgid ""
 "<figcaption>How Registered Servers work</figcaption>\n"
 "</figure>\n"
 msgstr ""
+"<figure>\n"
+"\t<img src=\"{% include img/de-screenshots/diagram-reg-server.inc %}\" loading=\"lazy\" alt=\"Diagramm: der Verbindungen zwischen Clients eines registrierten Jamulus-Servers\">\n"
+"<figcaption>So funktionieren registrierte /öffentliche Server</figcaption>\n"
```

Comment:
-->
`"<figcaption>So funktionieren registrierte/öffentliche Server</figcaption>\n"`

Reply:
Changed to "<figcaption>So funktionieren registrierte Server</figcaption>\n"
OK?

Reply:
Yea